src/cursor/abstract_cursor.ts

```diff
@@ -907,6 +904,13 @@ export abstract class AbstractCursor<
         'Unexpected null selectedServer. A cursor creating command should have set this'
       );
     }
+
+    if (!this.cursorSession) {
```

Comment:
kind of a nit, but any reason not to follow the `== null` pattern check here?

Reply:
No reason - fixed.